tasks/concurrent.js

```diff
@@ -4,16 +4,20 @@ var padStream = require('pad-stream');
 var async = require('async');
 var arrify = require('arrify');
 var indentString = require('indent-string');
+var chalk = require('chalk');
 
 var cpCache = [];
 
+var colors = ["blue","magenta","yellow","green","red","cyan","white","gray","redBright","greenBright","yellowBright","blueBright","magentaBright","cyanBright"];
```

Comment:
I think it's fine to use color on the 8 first tasks, and then just white on the rest.